dynamic_rest/renderers.py

```diff
@@ -0,0 +1,93 @@
+from collections import defaultdict
+from rest_framework import renderers
+
+
+class DynamicJSONRenderer(renderers.JSONRenderer):
+
+  def render(self, data, accepted_media_type=None, renderer_context=None):
+    # if data is a string, just render it out using the default method
+    # this usually indicates an error message
+    if isinstance(data, basestring):
+      return super(DynamicJSONRenderer, self).render(data, accepted_media_type, renderer_context)
+
```

Comment:
I think there should be another way to bypass this renderer (for example, for the output of metadata()). Also, some errors (from DRF?) are dicts that look like `{"detail":<error_message>}`. Maybe pass through if data is a `dict` but has no `_model`/`_pk` fields?